Dockerfile.arm

```diff
@@ -27,10 +27,28 @@ RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin"
 
 FROM multiarch/qemu-user-static:x86_64-arm as qemu
 FROM arm32v7/debian:buster-slim
+
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
+ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
+
 COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 RUN apt-get update \
- && apt-get install --no-install-recommends --no-install-suggests -y ffmpeg \
- libssl-dev ca-certificates \
+ && apt-get install --no-install-recommends --no-install-suggests -y ca-certificates gnupg curl && \
+ curl -s https://repo.jellyfin.org/debian/jellyfin_team.gpg.key | apt-key add - && \
+ curl -s https://keyserver.ubuntu.com/pks/lookup?op=get\&search=0x6587ffd6536b8826e88a62547876ae518cbcf2f2 | apt-key add - && \
```

Comment:
I'd rather pull this from debian than ubuntu but i dont know the keyserver

Reply:
Debian keyserver should be `keyring.debian.org`

Reply:
Thanks @Shadowghost but I still don't know the keyring link haha. I'm not too familiar with them.